templates/index.html

```diff
@@ -85,6 +90,14 @@ <h5 style="margin-bottom: 5%;">Equation Applied</h5>
             >
             {% csrf_token %}
             </form>
+
+            <div id="storage-manage-container"
+                 style="margin-top: 5%"
+            >
+                <p><button style="margin-top: 5%" type="button" onclick="savingData()" >Saving</button></p>
+                <input type="file" onchange="upload(this)" />
```

Comment:
Let's have "Saving" be an icon with a [tooltip](https://www.w3schools.com/css/css_tooltip.asp) that exports to JSON. Something like this: https://fontawesome.com/icons/file-export?style=solid

Reply:
In addition to this, let's have a [copy link icon](https://icons8.com/icons/set/copy-link) that copies the queryable link. It doesn't have to be anything special and can have a tooltip instead